src/browser/webview/mattermost.js

```diff
@@ -89,8 +72,57 @@ function overrideNotification() {
     var activeChannel = document.querySelector('.active .sidebar-channel').text;
     console.log(activeChannel);
     console.log(title);
-    if (activeChannel === title) {
-      ipc.sendToHost('onActiveChannelNotify');
+
+    // mentionCount for active channel
+    var newSeparators = document.getElementsByClassName('new-separator');
+    var post;
+    var isMentioned = false;
+    // Skip until real new-separator appear.
+    for (var i = 0; i < newSeparators.length; i++) {
+      if (newSeparators[i].offsetParent !== null) {
+        post = newSeparators[i];
```

Comment:
On Notification, `post` may be `null` depending on its timing on my environment (Mattermost 1.1). So I think it's better to manage only `notified` flag here, count mentions in interval timer.